From 07526e0d54567ac239d8f1fea6b1c82aa8194ae8 Mon Sep 17 00:00:00 2001
From: Sascha Mester <121475971+mettbroetchen43@users.noreply.github.com>
Date: Sat, 26 Oct 2024 17:27:29 +0200
Subject: [PATCH] hwdb: add Stream Deck Neo (#34903)

(cherry picked from commit f2eccaab343601ad687f17cd56201374fc603148)

Resolves: RHEL-49810
---
 hwdb.d/70-av-production.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/70-av-production.hwdb b/hwdb.d/70-av-production.hwdb
index 91c757bcb1..f7cc349822 100644
--- a/hwdb.d/70-av-production.hwdb
+++ b/hwdb.d/70-av-production.hwdb
@@ -71,6 +71,10 @@ usb:v0FD9p006D*
 usb:v0FD9p0080*
  ID_AV_PRODUCTION_CONTROLLER=1
 
+# Stream Deck Neo
+usb:v0FD9p009A*
+ ID_AV_PRODUCTION_CONTROLLER=1
+
 # Stream Deck Plus
 usb:v0FD9p0084*
  ID_AV_PRODUCTION_CONTROLLER=1
